app/assets/stylesheets/_user_journey.scss

```diff
@@ -0,0 +1,57 @@
+// Place all the styles related to the events controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
+
+.component-supporting-text{
+  font-weight: 300;
+}
+.govuk-details__code {
+  background-color: govuk-colour("light-grey");
+  border: 1px solid $govuk-border-colour;
+  padding: 0 15px;
+  margin-top: 15px;
+  overflow-x: auto;
+  white-space: pre-wrap;
+  word-wrap: break-word;
+}
+.secondary-link{
+  display: inline-block;
+  padding: 9px 10px 6px 10px;
+  font-family: "nta", Arial, sans-serif;
+  font-weight: 400;
+  font-size: 16px;
+  font-size: 1.1875rem;
+  line-height: 1.31579;
+}
+.secondary-link__confirmation{
+  padding: 9px 10px 6px 0px;
+}
+.certificate-details {
+  font-size: 0;
```

Comment:
what is this for?

Reply:
This is actually important for some reason as it reduces the margin above the boxes. All css was copied from the prototype as the design system doesn't really have anything similar. Will remove all the CSS that isn't necessary or is inherited